app/templates/components/build-message-line.hbs

```diff
@@ -1,9 +1,12 @@
 <div class="yml-message-line">
-  <div class="tooltip-wrapper">
+  <div class="tooltip-wrapper level-icon">
     <SvgImage @name={{this.level}} @class={{this.iconClass}} />
     <EmberTooltip @text={{this.tooltipText}} />
   </div>
   <div class="message">
     {{this.message}}
   </div>
+  <a href="{{config-get 'urls.buildConfigValidation'}}#{{this.code}}" class="message-help">
+    <SvgImage @name='icon-help' @class='icon-help' />
+  </a>
```

Comment:
I'd suggest to use [`ExternalLinkTo`](https://github.com/travis-ci/travis-web/blob/master/app/components/external-link-to.js) component here, which will automatically handle opening in new tab and address some potential security issues:
```
  <ExternalLinkTo href="{{config-get 'urls.buildConfigValidation'}}#{{this.code}}" class="message-help">
    <SvgImage @name='icon-help' @class='icon-help' />
  </ExternalLinkTo>
```

Reply:
done 👍